lib/clusterable/holding.rb

```diff
@@ -8,7 +8,35 @@ module Clusterable
   # A holding
   class Holding
     include EnumChron
+
+    ACCESSOR_ATTRS =
+      [
+        :ocn,
+        :local_id,
+        :enum_chron,
+        :n_enum,
+        :n_chron,
+        :n_enum_chron,
+        :status,
+        :condition,
+        :gov_doc_flag,
+        :mono_multi_serial,
+        :date_received,
+        :country_code,
+        :weight,
+        :uuid,
+        :issn
+      ].freeze
+
+    EQUALITY_EXCLUDED_ATTRS = [:uuid, :date_received].freeze
+    EQUALITY_ATTRS = (ACCESSOR_ATTRS - EQUALITY_EXCLUDED_ATTRS).freeze
```

Comment:
shouldn't `organization` also be in `EQUALITY_ATTRS`?

Reply:
you fixed this mid-review, so nevermind

Reply:
It's fixed now, in any case.